sdk/eventhub/azure-eventhubs/tests/asynctests/test_receive_async.py

```diff
@@ -296,3 +296,27 @@ def batched():
             assert list(message.body)[0] == "Event Data {}".format(index).encode('utf-8')
             assert (app_prop_key.encode('utf-8') in message.application_properties) \
                 and (dict(message.application_properties)[app_prop_key.encode('utf-8')] == app_prop_value.encode('utf-8'))
+
```

Comment:
Please add another test for the new `get_partition_properties` function.

Reply:
Added live test for get_partition_properties, get_partition_ids and get_partition_properties